src/annotator/config/settings.js

```diff
@@ -118,9 +118,10 @@ export default function settingsFrom(window_) {
     return jsonConfigs.query || queryFromURL();
   }
 
-  function hostPageSetting(name, options = {}) {
-    const hasDefaultValue = typeof options.defaultValue !== 'undefined';
-
+  /**
+   * @param {string} name - Unique name of the setting
```

Comment:
It would be useful to have a short summary of what sources this function consults and what it returns (the setting value if found or `undefined` otherwise).